modules/biofuels/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) , AUTHORS
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Please keep the original authors as they were, keeping the year, and add AUTHORS afterwards.

Something like: 

```
Copyright (c) 2018-2023 Tim Tröndle (ETH Zürich / IASS Potsdam), Bryn Pickering (ETH Zürich)
Copyright (c) 2024 AUTHORS
```